src/components/Search/Search.jsx

```diff
@@ -1,3 +1,5 @@
+import React from "react";
+import SearchButton from "./SearchButton/SearchButton";
```

Comment:
this should be ../SeachButton/search button since you are importing it from outside of the current folder